src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +10,16 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+	public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 123456";
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)